packages/xod-arduino-deploy/test/fixtures/packages/arduino/hardware/samd/boards.txt

```diff
@@ -368,3 +368,53 @@ tian_cons.pid.0=0xEA70
 tian_cons.descriptor.0=Standard Com Port
 tian_cons.hide=true
 tian_cons.build.board=SAMD_TIAN
+
+
+##############################################################
+# FOR TESTING OF MULTIPLE CPU MENU
+# NEXT LINES WAS COPIED FROM AVR ARCHITECTURE
+##############################################################
```

Comment:
~OF~
~WAS~ WERE

And I think the whole idea is fragile. Take samd… override with avr… a bit of mess.

Wouldn’t it better to simply take AVR txt or it’s part and place it in a separate fixture?